File: algorithm/src/main/java/baekjoon/p05575/Main.java
package baekjoon.p05575;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[][] inputs = new int[3][6];

        for (int i = 0; i < inputs.length; i++) {
            for (int j = 0; j < inputs[i].length; j++) {
                inputs[i][j] = sc.nextInt();
            }
        }

        for (int[] solve : solve(inputs)) {
            System.out.print(String.format("%d %d %d", solve[0], solve[1], solve[2]));
            System.out.println();
        }
    }

    private static int[][] solve(int[][] inputs) {
        int[][] ret = new int[3][3];

        for (int i = 0; i < inputs.length; i++) {
            int diff = (seconds(inputs[i][3], inputs[i][4], inputs[i][5]) - seconds(inputs[i][0], inputs[i][1], inputs[i][2]));
            ret[i][0] = diff / 3600;
            ret[i][1] = (diff % 3600) / 60;
            ret[i][2] = diff % 60;
        }

        return ret;
    }

    private static int seconds(int h, int m, int s) {
        return (3600 * h + 60 * m + s);
    }

}

File: algorithm/src/main/java/baekjoon/p05575/Main.kt
package baekjoon.p05575

fun main(args: Array<String>) {
    val inputs = List(3) { readLine()!!.split(" ").map { it.toInt() } }
    for (solve in solve(inputs)) {
        println(solve.joinToString(" "))
    }
}

private fun solve(inputs: List<List<Int>>): Array<Array<Int>> {
    val ret = Array(3) { Array(3) { 0 } }

    for (i in 0 until inputs.size) {
        val diff = seconds(inputs[i][3], inputs[i][4], inputs[i][5]) - seconds(inputs[i][0], inputs[i][1], inputs[i][2])
        ret[i][0] = diff / 3600
        ret[i][1] = (diff % 3600) / 60
        ret[i][2] = diff % 60
    }

    return ret
}

private fun seconds(h: Int, m: Int, s: Int): Int {
    return (3600 * h + 60 * m + s)
}